nixos/foundation.tt

```diff
@@ -1,29 +1,47 @@
 [% WRAPPER layout.tt title="Stichting NixOS Foundation" menu='nixos' %]
 [% PROCESS common.tt %]
 
-<h2>General</h2>
+<div class="row-fluid">
+<div class="span6">
+<h2>About The Foundation</h2>
+
+<p>The mission of the foundation is to support the Nix ecosystem's
+infrastructure, and projects implementing the purely functional
+deployment model.</p>
+
+<p>In particular, the foundation supports Nix, Nixpkgs, NixOS, and
+operates critical infrastructure:</p>
+
+<ul>
+  <li><a href="https://cache.nixos.org">cache.nixos.org</a> - the
+  NixOS binary cache, containing over 120TB of prebuilt Nix packages.</li>
+
+  <li><a href="https://hydra.nixos.org">hydra.nixos.org</a> - a Nix
+  build farm, consisting of hundreds of macOS, x86-64 (Linux) and
+  aarch64 (Linux) build cores. Hydra executes over 350,000 builds each
+  week.</li>
+</ul>
 
 <p>The NixOS Foundation is a registered non-profit organisation at the Chamber of
 commerce (Kamer van Koophandel) in Utrecht, The Netherlands. The KvK number is 63520583.
 </p>
 
-<p>The purpose of the foundation, is to help support the infrastructure and projects
-that are implementing the purely functional deployment model, such as NixOS, Nix
-and NixOps.</p>
-
-<p>There are currently 3 board members:</p>
+<p>There board member consists of:</p>
```

Comment:
> There board member

I don't have strong opinions about a replacement, but I do know this isn't cromulent english.

Reply:
Thanks, fixed